### ISSUE TITLE
Unable to find IP unit conversions for cm and Yes/No

### ISSUE THREAD

**Issue by JasonGlazer** [NREL/EnergyPlus#6102]
When running the test file, the following warning messages are shown:

```
   ** Warning ** Unable to find a unit conversion from Inside Height [cm] into IP units
   **   ~~~   ** Applying default conversion factor of 1.0
   ** Warning ** Unable to find a unit conversion from Near Radiant [Yes/No] into IP units
   **   ~~~   ** Applying default conversion factor of 1.0
   ** Warning ** Unable to find a unit conversion from Has Active HVAC [Yes/No] into IP units
   **   ~~~   ** Applying default conversion factor of 1.0

```

These should be corrected.

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (see EnergyPlusDevSupport\DefectFiles\6000s\6102)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
These are all in the Surface Convection Parameters table of the Initialization Summary (eio) report.

![image](https://cloud.githubusercontent.com/assets/4991075/25141484/4ddaf894-2429-11e7-91d3-7a476dac1628.png)

**Comment by JasonGlazer** [comment]
This issue must have been fixed between 8.7.0 and 9.1.0 because when the file is updated it no longer generates the warning message.
